logging.md

```diff
@@ -0,0 +1,34 @@
+# Conventions for LOGGING
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All logging may be stored in files. 
+If so, they must be given in Provenance as a result. 
+
+## Log Levels
+A more detailed description will be found [here](https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels/5278006#5278006)
+### TRACE / ALL
+This is the most verbose logging level to trace the path of the algorithm.
+E.g.: Start/End/Duration of a method, even loops may be logged. 
+Note that logging within loops can dramatically affect performance.
+### DEBUG
+This level is used if parameters and or the status of an algorithm/method should be logged. 
+### INFO
+Log information about used settings on application level.
+### WARN
+This level is used to log information about missing/wrong configurations which may lead to errors.
+### ERROR
+Log all events that produce no or a wrong result. 
+It is useful to output all the information that helps to determine the cause without the need for further investigation.
+
+
+## Format
+The format of the logging output has to be formatted like this:
+TIMESTAMP LEVEL LOGGERNAME - MESSAGE
+***Example:***
+08:03:40.017 WARN edu.kit.ocrd.MyTestClass - A warn message
+:warning: 
+Since only the time stamp is logged, the log should be written to files that change daily.
```

Comment:
IIUC, you want logging to be the outcome of a job/task/process right (because of the bit about provenance at the top). That would not be a continuous runnign process where log rotation was important. Or do you want to concatenate the logging files of processes into a big file?

Reply:
This is a clumsy phrase. The output should never be collected in a large file. The statement should be that the output can always be assigned to a work step.